source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,117 @@
+.. _swupd_mixer_integration:
+
+Adding custom packages and bundles
+##################################
+
+
+
+Prerequisites
+=============
+
+To use the new swupd mixer feature, you'll need a recent image of Clear
+Linux OS for Intel Architecture with the following bundle installed. If you
+don't have it already, you can add it with the :command:`swupd bundle-add`
+command like so::
+
+  # swupd bundle-add mixer
+
+What is it
+================
+The software updater now has native integration with the mixer tool. One of the
+highest requests for Clear Linux OS for Intel Architecture was the ability for
+users to add their own custom content to their OS. Mixer initially enabled this,
+but it required users to spin their own full OS, and be the OSV for it making
+all the releases and update content.
```

Comment:
> Mixer initially enabled this, but it required users to spin their own full OS, and be the OSV for it making all the releases and update content.

Recommend "Prior to this, it required users to spin their own full OS, and be the OSV using the Mixer tool."